src/ReactionMechanismSimulator.jl

```diff
@@ -2,30 +2,23 @@ module ReactionMechanismSimulator
 ENV["JULIA_CONDAPKG_BACKEND"] = "MicroMamba"
```

Comment:
Is it appropriate/necessary to be force-modifying the environment variable inside the script here? Doesn't it means the user has no control over `$JULIA_CONDAPKG_BACKEND` ?